imports/plugins/included/shipping-rates/server/no-meteor/schemas/restrictions.graphql

```diff
@@ -0,0 +1,217 @@
+####
+# Custom data types for FlatRateFulfillmentRestriction data
+####
+type attributeRestrictions {
+  property: String
+  value: String
+  propertyType: String
+  operator: String
+}
+
+type destinationRestrictions {
+  country: [String]
+  region: [String]
+  postal: [String]
+}
+
+enum restrictionTypeEnum {
+  allow
+  deny
+}
+
+enum FlatRateFulfillmentRestrictionSortByField {
+  createdAt
+}
+
+input attributeRestrictionsInput {
+  property: String
+  value: String
+  propertyType: String
+  operator: String
+}
+
+input destinationRestrictionsInput {
+  country: [String]
+  region: [String]
+  postal: [String]
+}
+
+
+####
+# Queries
+####
+extend type Query {
+  "Get the full list of flat rate fulfillment method restrictions."
+  getFlatRateFulfillmentRestrictions(
+    shopId: ID!
+    after: ConnectionCursor,
+    before: ConnectionCursor,
+    first: ConnectionLimitInt,
+    last: ConnectionLimitInt,
+    sortOrder: SortOrder = desc,
+    sortBy: FlatRateFulfillmentRestrictionSortByField = createdAt
+  ): FlatRateFulfillmentRestrictionConnection!
+
+  getFlatRateFulfillmentRestriction(
+    shopId: ID!
+    restrictionId: ID!
+    after: ConnectionCursor,
+    before: ConnectionCursor,
+    first: ConnectionLimitInt,
+    last: ConnectionLimitInt,
+    sortOrder: SortOrder = desc,
+    sortBy: FlatRateFulfillmentRestrictionSortByField = createdAt
+  ): FlatRateFulfillmentRestrictionConnection!
+}
+
+type FlatRateFulfillmentRestrictionConnection {
+  edges: [FlatRateFulfillmentRestrictionEdge]
+  nodes: [FlatRateFulfillmentRestriction]
+  pageInfo: PageInfo!
+  totalCount: Int!
+}
+
+type FlatRateFulfillmentRestrictionEdge {
+  cursor: ConnectionCursor!
+  node: FlatRateFulfillmentRestriction
+}
+
+"Defines a restriction for a single flat rate fulfillment method, or universally all flat rate fulfillment methods."
+type FlatRateFulfillmentRestriction implements Node {
+  "The restriction ID."
+  _id: ID!
+
+  "The shop ID"
+  shopId: ID!
+
+  "Method IDs to apply this restriction to."
+  methodIds: [String]
+
+  """
+  The type of this restriction. Allowed types are "allow" or "deny".
+  """
+  type: restrictionTypeEnum!
+
+  "Attribute restrictions."
```

Comment:
Clarify in description whether multiple are evaluated with AND or OR, and whether they combine with destination restrictions with AND or OR

Reply:
Also update the descriptions in the `input` version of this type to match